packages/orchestration/src/examples/send-anywhere.flows.js

```diff
@@ -51,14 +52,20 @@ export const sendIt = async (
 
   await localTransfer(seat, contractState.localAccount, give);
 
-  await contractState.localAccount.transfer(
-    { denom, value: amt.value },
-    {
-      value: destAddr,
-      encoding: 'bech32',
-      chainId,
-    },
-  );
+  try {
+    await contractState.localAccount.transfer(
+      { denom, value: amt.value },
+      {
+        value: destAddr,
+        encoding: 'bech32',
+        chainId,
+      },
+    );
+  } catch (e) {
+    await withdrawToSeat(contractState.localAccount, seat, give);
+    throw seat.fail(makeError(`IBC Transfer failed ${q(e)}`));
```

Comment:
`seat.fail()` returns nothing. I see that [our docs suggest this](https://docs.agoric.com/reference/zoe-api/zcfseat.html#azcfseat-fail-msg), but it doesn't make any sense to me.

I see the pattern in some of our example contracts, but not our production contracts.

Reply:
What would you expect to see instead?

With:
```js
  seat.fail(makeError(`IBC Transfer failed ${q(e)}`));
  return;
```
I see **Promise resolved with `undefined`.**

With:
```js
 seat.fail(makeError(`IBC Transfer failed ${q(e)}`));
 ```
Execution continues and the `seat.exit()` a few lines down causes a rejection with `Error: seat has been exited`.

_I'm using `yarn test test/examples/send-anywhere.test.ts -m 'failed ibc transfer returns give'` to hit this path._

Reply:
Another option considered a while back was:
```js
  seat.exit();
  throw makeError(`IBC Transfer failed ${q(e)}`) 
```
but this seemed like an anti-pattern since `seat.fail` exists

Reply:
I would have gone with 
```
  const errorMsg = `IBC Transfer failed ${q(e)}`;
  seat.exit(errorMsg);
  throw  makeError(errorMsg);
```